src/lib/util.js

```diff
@@ -136,7 +140,11 @@ export function createPrefetch(url, preload = false) {
     link.href = url;
 
     if (preload) {
-        link.as = url.indexOf('.js') !== -1 ? 'script' : 'style';
+        link.as = url.indexOf('.css') !== -1 ? 'style' : 'script';
```

Comment:
Minor, but should we keep this as-is and add `.mjs` as a second condition, instead?